docs/git-branching-strategy.md

```diff
@@ -0,0 +1,126 @@
+(see the CI/CD process for more details), it must *always contain a working version of the site*, with the latest fixes
+and developed features.
+
+When developing a new feature, a _feature branch_ is created from `main`. A _feature branch_ is generally named
+`<TICKET>-<DESCRIPTION>` where `<TICKET>` is a related ticket number (ex: a Jira or Basecamp ticket) and `<DESCRIPTION>`
+is a short description of the feature. See (B) in the graph below for an example.
+
+The _feature branches_ must be short-lived (no more than a few days) and must be limited in the size of the feature they
+introduce. Large features should be split into smaller features. We want to prevent having long-lasting branches that
+become difficult to merge.
+
+Merging a _feature branch_ into `main` must pass through the Pull Request process. Always use "Squash and Merge" when
+merging.
+
+![Single main branch and feature branches](images/git-01-main-branch.svg)
+
+
+## Creating a release
+
+A "release" is a version of the code that is considered ready for production. In this Git strategy, a new release is
+always for a version containing _new features_. For a version containing only bug fixes, see the "Hot fixes" section
+below.
+
+When a release is ready to be made, two steps must be taken:
+
+1. **On the commit on `main` that you want to release, add a _release tag_.**
+   
+   A _release tag_ serves to mark a commit containing a released version. A _release tag_'s name is the release's
+   version number `<MAJOR>.<MINOR>.<PATCH>` preceded by `v`. Example: `v2.1.0`. See (C) in the graph below. 
+2. **From the same commit, create a new _release branch_.**
+
+   A _release branch_ allows to eventually implement hot fixes in case a bug is discovered in this release (see
+   the ["Hot fixes" section](#hot-fixes) below). The _release branch_'s name is `release/<MAJOR>.<MINOR>.x`. Note that
+   `<PATCH>` is replaced by `x`. Example: `release/2.1.x`. See (D) in the graph below. 
+
+![Creating a release](images/git-02-release.svg)
+
+It's possible to create a release from a previous commit if we don't want the latest features of `main` to be
+released yet. See (E) in the graph below.
+
+Development of new features continue on the `main` branch after the released commit.
+
+![Creating a release on a previous commit](images/git-03-release-before.svg)
+
+When another release from the `main` branch is ready to be created, a new _release tag_ and _release branch_ are
+created on the commit to release. See (F) in the graph below for an example for version 2.2.0.
+
+![Creating a new release](images/git-04-new-release.svg)
+
+### Release version number
+
+The version number for the release is determined by the developer, based on the previous release's version number. It is
+globally inspired by the [semver](https://semver.org/) format `<MAJOR>.<MINOR>.<PATCH>`.
+
+* The `<PATCH>` is increased when introducing fixes that don’t introduce or change existing features. In this Git
```

Comment:
Maybe explicitly specify that the PATCH number is only for hot fixes, with a link to the hot fix section below.